Sources/Wrappers/AngouriMath.FSharp/Matrices.fs

```diff
@@ -3,45 +3,93 @@
 open AngouriMath
 open AngouriMath.FSharp.Core
 open AngouriMath.FSharp.Functions
+open System.Linq
 
-let (|*) a b = (parsed a * parsed b).InnerSimplified |> asMatrix
-
-let (|**) a b = (parsed a ** parsed b).InnerSimplified |> asMatrix
-
-let (|+) a b = (parsed a + parsed b).InnerSimplified |> asMatrix
-
-let (|-) a b = (parsed a - parsed b).InnerSimplified |> asMatrix
-
-let (|/) a b = (parsed a / parsed b).InnerSimplified |> asMatrix
+/// Creates a matrix from a list of lists
+/// of objects (which are parsed into Entities)
+let matrix x = 
+ let rec columnCount (x : 'T list list) =
```

Comment:
Oopsie I smell an identation problem here. There are a few more down in the file too